### PR TITLE
1563 publish move send onto its own page in the publish flow and change to radio buttons

### PR DESCRIPTION
**Pull request by elceebee**
### Context
Improvements to publish before the next recruitment cycle

### Changes proposed in this pull request
Replaces the SEND check box with Yes / No Radio buttons. 
Originally, we were going to make this into a separate page, but it was decided that it would remain on the level page. (see trello card)

| Before    | After  |
| -------- | ------- |
| image |  image |

https://github.com/DFE-Digital/publish-teacher-training/assets/44073106/4b5ba9c9-0629-4cec-9050-5c3eba56b2c9

### Trello card
https://trello.com/c/Rkqty0mi

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally
- [NA] Inform data insights team due to database changes

### CODE REVIEW

**Comment by elceebee** [comment]
[Publish: Move SEND onto it's own page in the publish flow, and change to radio buttons](https://trello.com/c/Rkqty0mi/1563-publish-move-send-onto-its-own-page-in-the-publish-flow-and-change-to-radio-buttons)

**Review by avinhurry — COMMENTED** [review]
Hmm, I don't think we should be selecting radio buttons for users 

> Do not pre-select radio options as this makes it more likely that users will:
>
> - not realise they’ve missed a question
> - submit the wrong answer

https://design-system.service.gov.uk/components/radios/

Do you have any thoughts on this @kelliedesigner?

**Comment by elceebee** [comment]
> Hmm, I don't think we should be selecting radio buttons for users
> 
> > Do not pre-select radio options as this makes it more likely that users will:
> > 
> > * not realise they’ve missed a question
> > * submit the wrong answer
> 
> https://design-system.service.gov.uk/components/radios/
> 
> Do you have any thoughts on this @kelliedesigner?

Hi Avin, I raised this as a question in the trello ticket because at some point, we decided that the field should have a default value of false.

File: spec/features/publish/courses/new_level_spec.rb [hunk 1]
```diff
@@ -97,18 +97,18 @@ def then_i_am_met_with_errors
   end
 
   def then_with_send_is_in_params
-    expect(page.current_url).to match(/is_send%5D=1/)
+    expect(page.current_url).to match(/is_send%5D=true/)
   end
 
   def primary_level_selected_params
-    '?course%5Bis_send%5D=0&course%5Blevel%5D=primary'
+    '?course%5Bis_send%5D=false&course%5Blevel%5D=primary'
```
Comment (dcyoung-dev) [inline spec/features/publish/courses/new_level_spec.rb:104]: 😍

**Review by dcyoung-dev — APPROVED** [review]
Code looks spot on - I've not had time to pull this down locally

**Review by avinhurry — APPROVED** [review]
Nice work, looks good!

**Review by dcyoung-dev — APPROVED** [review]

**Pull request merged**